### PR TITLE
Add xp command

### PR DESCRIPTION
**Pull request by gaoyichuan**
Add support for command "xp".  Which is a command for give the given player the given number or levels of exprience. Tested on my server

### CODE REVIEW

File: src/pocketmine/lang/locale/eng.ini [hunk 1]
```diff
@@ -129,8 +129,8 @@ commands.seed.success=Seed: {%0}
 
 commands.bancidbyname.success=Banned player {%0}'s CID
```
Comment (xpyctum) [inline src/pocketmine/lang/locale/eng.ini]: **break** github syntax

**Comment by gaoyichuan** [comment]
I will work on summon command the following two days, since it's a bit complex for me xD.

File: src/pocketmine/lang/locale/eng.ini (hunk 1 above)
Comment (gaoyichuan) [inline src/pocketmine/lang/locale/eng.ini]: @xpyctum What?

Comment (xpyctum) [inline src/pocketmine/lang/locale/eng.ini]: @gaoyichuan symbol ' is break github syntax

**Comment by gaoyichuan** [comment]
@xpyctum These lines are NOT added by me, and I think there aren't any problems since they just break the syntax of github, not the project.

**Comment by gaoyichuan** [comment]
@PocketKiller Why close it?

**Comment by PeratX** [comment]
@gaoyichuan He thinks you forgot register the command. Please check it :)

**Comment by PocketKiller** [comment]
hmm I haven't seen that line I am completely blind.

**Pull request merged**